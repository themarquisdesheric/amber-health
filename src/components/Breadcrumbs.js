import React from 'react';
import PropTypes from 'prop-types';
import { Link } from 'gatsby';

const Breadcrumbs = ({ path }) => (
  <p className={`${path}-breadcrumbs`} style={{ marginBottom: '3rem' }}>
    <Link to="/">
      Home
    </Link>
    {' â†’ '} 
    <Link to={`/${path.toLowerCase()}`}>
      {path.replace('/', '')}
    </Link>
  </p>
);

Breadcrumbs.propTypes = {
  path: PropTypes.string.isRequired
};

export default Breadcrumbs;
