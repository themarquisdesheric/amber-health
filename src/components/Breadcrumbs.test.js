import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

import Breadcrumbs from './Breadcrumbs';

test('Breadcrumbs', () => {
  const wrapper = shallow(<Breadcrumbs path="Articles" />);

  expect(toJson(wrapper)).toMatchSnapshot();
});
