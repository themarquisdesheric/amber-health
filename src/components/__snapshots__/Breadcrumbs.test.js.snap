// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Breadcrumbs 1`] = `
<p
  className="breadcrumbs"
>
  <mockConstructor
    to="/"
  >
    Home
  </mockConstructor>
   â†’ 
  <mockConstructor
    to="/articles"
  >
    Articles
  </mockConstructor>
</p>
`;
